controllers/searchController.js

```diff
@@ -0,0 +1,10 @@
+const searchService = require("../services/searchService");
+const { catchAsync } = require("../utils/error");
+
+const getSearchResults = catchAsync(async (req, res) => {
+  const queryParams = req.query.keyword;
```

Comment:
검색 api에서는 keyword라는 이름의 query string만 받을 것이기에 변수이름도 queryParams보다는 keyword라고 해주시는게 명확합니다. 또한 req.query.keyword라고 dot notation을 쓰기보다는 구조분해 할당을 사용해주세요.
```suggestion
  const { keyword } = req.query;
```